backend/app/schemas/interview.py

```
@@ -0,0 +1,19 @@
+from typing import Literal
+
+from pydantic import BaseModel
+
+
+# 会話の1ターン。履歴はクライアントが保持し、毎回全部送ってくる（サーバーは保存しない）
+class Turn(BaseModel):
+    role: Literal["assistant", "user"]
+    content: str
+
+
+# ---- POST /interviews/chat（次の質問を生成）----
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)